src/appComponents/tracks/TracksController.js

```diff
@@ -15,29 +17,28 @@ tracksModule.config(["$stateProvider", function($stateProvider) {
 
 }]);
 
-tracksModule.controller("TracksController", ["$mdDialog","$sessionStorage", "$rootScope", "ApiJsonFactory",
+tracksModule.controller("TracksController", ["$mdDialog", "$sessionStorage", "$rootScope", "ApiJsonFactory",
         function ($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
-          var tsc = this;
+          let tsc = this;
 
-          if ($sessionStorage.tracks === null || 
-            typeof $sessionStorage.tracks === "undefined")
+         if ($sessionStorage.tracks === null || typeof $sessionStorage.tracks === "undefined")
          {
 
             $sessionStorage.tracks = [];
-        }
-        tsc.Tracks = $sessionStorage.tracks;
-        tsc.Sessions = $sessionStorage.sessions;
-        
-        if (tsc.Tracks.length === 0) {
+
+         }
+          tsc.Tracks = $sessionStorage.tracks;
+          tsc.Sessions = $sessionStorage.sessions;
+          if (tsc.Tracks.length === 0) {
 
           ApiJsonFactory.getJson("tracks")
             .then(function (response) {
 
             tsc.Tracks = response.data.tracks;
             $sessionStorage.tracks = tsc.Tracks;
                 
-            }, function error (error) {
-                    //console.error(error);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['error' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2431821597/issues/source?bid=3233857&fileBranchId=3238420#l40)